package/MDAnalysis/__init__.py

```diff
@@ -167,18 +169,18 @@
 
 # Registry of Readers, Parsers and Writers known to MDAnalysis
 # Metaclass magic fills these as classes are declared.
-_READERS = {}
-_READER_HINTS = {}
-_SINGLEFRAME_WRITERS = {}
-_MULTIFRAME_WRITERS = {}
-_PARSERS = {}
-_PARSER_HINTS = {}
-_SELECTION_WRITERS = {}
-_CONVERTERS = {}
+_READERS: Dict = {}
+_READER_HINTS: Dict = {}
+_SINGLEFRAME_WRITERS: Dict = {}
+_MULTIFRAME_WRITERS: Dict = {}
+_PARSERS: Dict = {}
+_PARSER_HINTS: Dict = {}
+_SELECTION_WRITERS: Dict = {}
+_CONVERTERS: Dict = {}
 # Registry of TopologyAttributes
-_TOPOLOGY_ATTRS = {}  # {attrname: cls}
-_TOPOLOGY_TRANSPLANTS = {}   # {name: [attrname, method, transplant class]}
-_TOPOLOGY_ATTRNAMES = {}  # {lower case name w/o _ : name}
+_TOPOLOGY_ATTRS: Dict = {}   # {attrname: cls}
+_TOPOLOGY_TRANSPLANTS: Dict = {}   # {name: [attrname, method, transplant class]}
+_TOPOLOGY_ATTRNAMES: Dict = {}   # {lower case name w/o _ : name}
```

Comment:
@umak1106 It would be great to have these lines in a separate PR so we can work at annotating other parts of the code without triggering errors here.

Reply:
ok mentor